src/demo/demo.ejs

```diff
@@ -47,6 +47,8 @@
         }
       }
     </style>
+    <link href="style.css" rel="stylesheet" />
+    <script src="onfidoAuth.min.js"></script>
```

Comment:
Wouldn't `onfido.min.js` need to be included also?

Reply:
No, because `Onfido` is already injected into the `window` by webpack. Anyways, this is PR broken at the moment. For some reason the surge demo app does not seem to be using the latest code I've pushed. I won't have time to look into this before my sabbatical so I will leave it with BioExp.
What's needed to make this work is making sure that `PUBLIC_PATH` for surge deployment is `'/'` instead of the S3 assets path. I added this command `"build:deployment": "TEST_ENV=deployment NODE_ENV=test npm run build && npm run build:auth"` to achieve that but I can see that  the `PUBLIC_PATH` is still pointing at the S3 asset URL. It might be because `NODE_ENV` in the `build:deployment` command is overridden by `matrix.NODE_ENV` in GitHub Actions. @carlagn FYI ^